examples/js-npm-typescript/src/analytics/index.ts

```diff
@@ -2,21 +2,6 @@
  * This client was automatically generated by RudderTyper. ** Do Not Edit **
  */
 
-/**
- * Ajv is a peer dependency for development builds. It's used to apply run-time validation
```

Comment:
Don't we want to use dev builds for examples?

Reply:
Dev build doesn't fire events while running the sample apps it only does validation. Should we update all to prod build?

Reply:
Update all the examples to the prod build as they only fire events while running the sample app